pkg/apis/policy/validation/validation.go

```diff
@@ -27,13 +27,14 @@ import (
 )
 
 func ValidatePodDisruptionBudget(pdb *policy.PodDisruptionBudget) field.ErrorList {
-	allErrs := ValidatePodDisruptionBudgetSpec(pdb.Spec, field.NewPath("spec"))
+	allErrs := apivalidation.ValidateObjectMeta(&pdb.ObjectMeta, true, ValidatePodDisruptionBudgetName, field.NewPath("metadata"))
```

Comment:
Tightening validation on existing objects breaks upgrade and migration

Reply:
thanks